test/utils/navigation.spec.ts

```diff
@@ -37,11 +38,10 @@ describe("NAVIGATION utils", () => {
   });
 
   test(`getRemoveBackLink returns a URL with the 'journey' query parameter present when on the Remove journey`, () => {
-    const mockRequest = { } as Request;
-    mockRequest["query"] = {
+    mockRemoveRequest["query"] = {
```

Comment:
Can these three lines also move up and be removed here and below?

 ```
   mockRequest["query"] = {	
      "journey": "remove"	
    };	
```

Reply:
Moved these up as well.